### ISSUE TITLE
Grimoire Generator Name 

### ISSUE THREAD

**Issue by aturfah** [aturfah/eou-grimoire-editor#1]
```
this is true for both classic and story mode

00 09 08 01 0C 00 82 62 82 85 82 89 82 92 82 8F 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 84 00 0A 00 87 00 05 00 88 00 05 00 7D 00 08 00 32 00 0A 00 16 00 05 00 34 00 03 00 

for example... 82 62 82 85 82 89 82 92 82 8F is the name
```

The "generator" is the name of the person who made the grimoire; watch out for story mode indicators in the class hex as well

```
00 02 Protector (Shield)
08 02 Protector (Shield) - Raquna
```

**Comment by aturfah** [comment]
Some notes
- Flag has to be set for 'unknown origin', this will override the name provided
- Names use the full-width characters; be mindful of going back & forth because these are not ASCII compatible
- King's Grimoires will use the name provided

**Comment by aturfah** [comment]
Addressed in #2
